pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -867,9 +867,9 @@ protected void internalCreateSubscription(String subscriptionName, MessageIdImpl
                 if (topic.getSubscriptions().containsKey(subscriptionName)) {
                     throw new RestException(Status.CONFLICT, "Subscription already exists for topic");
                 }
-
+                //TODO support dead letter topic in admin
```

Comment:
Please open another issue to tracking this?

Reply:
It's not useful any more because client subscribe can update the max redelivery count and dead letter topic name. I will delete the TODO.